src/VirtoCommerce.Platform.Web/Controllers/Api/SecurityController.cs

```diff
@@ -539,9 +539,39 @@ public async Task<ActionResult> RequestPasswordReset(string loginOrEmail)
         [HttpPost]
         [Route("validatepassword")]
         [Authorize]
-        public async Task<ActionResult<PasswordValidationResult>> ValidatePassword([FromBody] string password)
+        public async Task<ActionResult<IdentityResult>> ValidatePassword([FromBody] string password)
```

Comment:
Mark this method as Obsolete with a comment to use   other version instead, and please and hide this method from SwaggerUI

Reply:
the old PasswordValidationResult class was deleted, no way to keep it.